python/lsst/meas/algorithms/cloughTocher2DInterpolator.py

```diff
@@ -129,14 +135,20 @@ def run(
             ctUtils.updateArrayFromImage(goodpix, maskedImage.image)
 
         # Construct the interpolant with goodpix.
+        if self.config.flipXY:
+            points = list(zip(goodpix[:, 1], goodpix[:, 0]))
+        else:
+            points = list(zip(goodpix[:, 0], goodpix[:, 1]))
+
         interpolator = CloughTocher2DInterpolator(
-            list(zip(goodpix[:, 0], goodpix[:, 1])),
+            points,
             goodpix[:, 2],
             fill_value=self.config.fillValue,
         )
 
         # Compute the interpolated values at bad pixel locations.
-        badpix[:, 2] = interpolator(badpix[:, :2])
+        col_slice = slice(1, None, -1) if self.config.flipXY else slice(2)
```

Comment:
I think this would be easier to read with a regular `if` block and two calls to `interpolator` with what's currently `col_slice` written out explicitly inside the square brackets.